pyproject.toml

```diff
@@ -26,36 +24,44 @@ dependencies = [
     "matplotlib>=3.7.1",
     "pandas>=2.0.2",
     "attrs>=23.1.0",
+    "PyYAML>=6.0.1",
 ]
-dynamic = ["version", "readme"]
+dynamic = ["version"]
 
+[project.urls]
+"Homepage" = "https://github.com/SpectroInlets/spectro_inlets_quantification"
+"Bug Tracker" = "https://github.com/SpectroInlets/spectro_inlets_quantification/issues"
 
 [project.optional-dependencies]
-dev = [
-    "invoke",
-    "rich",
-    "black==23.3.0",
-    "ruff==0.0.270",
-    "mypy==1.3.0",
-    "pytest==7.3.1",
-    "sphinx",
-    "myst-parser",
-    "sphinx-rtd-theme",
-    "git+https://github.com/executablebooks/sphinx-proof@d37495f",
-    "freezegun",
-    "toml",
-]
 rest = ["docutils>=0.3", "pack ==1.1, ==1.3"]
 
+###########################
+# Packaging configuration #
+###########################
+
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
 [tool.setuptools.packages.find]
 where = ["src"]
 
 [tool.setuptools.dynamic]
 version = {attr = "spectro_inlets_quantification.__version__"}
-# readme = {file = ["README.md"]}
-# The readme prevents uploading because PyPI expects rst. I specified in setup.py that it is
-#   markdown, but can't find the syntax to do it here. See:
-#   https://packaging.python.org/en/latest/specifications/core-metadata/#description-content-type
+# readme = {file = ["README.md"], content-type = "text/markdown"}
+
+[tool.setuptools.package-data]
+your_package_name = [
```

Comment:
This is fine, but isn't there a way to include any .yml file in data, however many subfolders deep it is?

Reply:
Yes there is, good catch